src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const BASE_COST = 40;
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    return days * BASE_COST - SHORT_TERM_DISCOUNT;
```

Comment:
The logic here applies a fixed discount for short-term rentals. Ensure that this is the intended behavior, as it might be more common to apply a percentage discount rather than a fixed amount.